apps/web/src/components/SearchRequestTable/SearchRequestTableClient.tsx

```diff
@@ -167,18 +167,18 @@ export const SearchRequestTable = ({
   );
 };
 
-const SearchRequestTableApi = ({ title }: { title: string }) => {
+const SearchRequestTableWrapper = ({ title }: { title: string }) => {
   const [result] = useGetPoolCandidateSearchRequestsQuery();
   const { data, fetching, error } = result;
 
   return (
     <Pending fetching={fetching} error={error}>
-      <SearchRequestTable
+      <SearchRequestTableClient
         poolCandidateSearchRequests={data?.poolCandidateSearchRequests ?? []}
         title={title}
       />
     </Pending>
   );
 };
 
-export default SearchRequestTableApi;
+export default SearchRequestTableWrapper;
```

Comment:
Calling the wrapper around a component named `_____` the `_____Api` component is a pattern we use, but confusing in this case. Given this file will be deleted before long, it should be palatable. 😛